docs/src/pages/[...docPath].tsx

```diff
@@ -74,8 +74,8 @@ export const getStaticProps: GetStaticProps = async (props) => {
         seoDescription,
       },
       layout,
-      next,
-      prev,
+      next: data.next ?? next,
```

Comment:
do we allow only `next: false` in the frontmatter? what if user enters `next: true` - would this work in this case?

Reply:
@RavenInd

Reply:
corrected